internal/host/hostcommands/install_cmd.go

```diff
@@ -248,14 +248,13 @@ func (i *installCmd) getDiskUnbootableCmd(ctx context.Context, host models.Host)
 	for _, disk := range inventory.Disks {
 		isFcIscsi := strings.Contains(disk.ByPath, "-fc-") || strings.Contains(disk.ByPath, "-iscsi-")
 		if disk.Bootable && !isFcIscsi && !disk.IsInstallationMedia {
-			dev := hostutil.GetDeviceFullName(disk.Name)
-			formatCmds += fmt.Sprintf("dd if=/dev/zero of=%s bs=512 count=1 ; ", dev)
+			formatCmds += fmt.Sprintf("dd if=/dev/zero of=%s bs=512 count=1 ; ", disk.ID)
```

Comment:
Why `disk.ID`? this should be `disk.Path`

Reply:
Add a test where the installation_disk_path is set to path and see how do you handle the conversion. This will also test the upgrade scenario

Reply:
@omertuc The disk id is a symlink to the disk path so it should be fine. I will test it soon.
@slaviered can you please provide more details about your comment? which conversion?

Reply:
set installation_disk_id (now that we have new field) to /dev/sda and see that it changed to the newer ID